src/main/webapp/css/main.css

```diff
@@ -14,26 +14,68 @@
  * limitations under the License.
  */
 
-/* Remove default styling from the body tag. */
-body {
-  padding: 0;
-  margin: 0;
+/* Title animation*/
+.border{
+    background: linear-gradient(110deg, #EBCB8C 33%, rgba(0, 0, 0, 0) 33%), linear-gradient(110deg, #ffb972 34%, #88D8B0 34%);
+    background-size: 300% 300%;
+    background-position: 25% 50%;
+    padding: 5px;
+    height: 300px;
+    -webkit-animation: Gradient 15s ease infinite;
```

Comment:
It should be possible to use this unprefixed:
https://developer.mozilla.org/en-US/docs/Web/CSS/animation